### ISSUE TITLE
resolve combos/troupes confusion

### ISSUE THREAD

**Issue by jonkiparsky** [bethlakshmi/GBE2#738]
Apparently at some point there was an attempt to remove one or the other of combo or troupe, which ended up with massive confusion around these classes in gbe/views, in that there is a "create_combo" function, but no "edit_combo", and there is "edit_troupe" but no "create_troupe" 

Definition of "done": 
1. migrate "combos" to "troupes" 
2. remove "combo" from models
3. resolve any resulting breakage
4. ensure that troupe create/edit/review/etc all works. 
5. ensure that any new code has 100% coverage

**Comment by burlexpo** [comment]
Related (although I can make this a separate issue if you want) -- once this is done, document how to create a troupe as a user and review the help text, because currently it's really confusing.

**Comment by bethlakshmi** [comment]
eliminated combos.  Confusion resolved.
